app/views/landing_page/blocks/_heading.html.erb

```diff
@@ -0,0 +1,10 @@
+<%
+  heading_level = block.data["heading_level"] || 2
+  padding = block.data["padding"] || false
+%>
+
+<%= render "govuk_publishing_components/components/heading", {
+  text: block.data["text"],
+  heading_level: heading_level,
+  padding: true
```

Comment:
I think this should be `padding:` to pick up the value in `block.data["padding"]`, otherwise it will always be set to `true`.

Is there a time when the padding shouldn't be set? I'm wondering if it should always be true in the heading block?

Reply:
@leenagupte Good spot, thanks. I will remove the `.yaml` option for now.